src/matrand_model.jl

```diff
@@ -0,0 +1,82 @@
+export mat_rand_model
+
+using Noise
+
+function mat_rand(m::Int, n::Int, r::Int, sr::Float64, va::Float64, vb::Float64, c::Float64)
+  xl = rand(Uniform(-0.1, 0.3), m, r)
+  xr = rand(Uniform(-0.1, 0.3), n, r)
+  xs = xl * xr'
+  Ω = findall(<(sr), rand(m, n))
```

Comment:
I suggest to vectorize Ω
```suggestion
  Ω = findall(<(sr), rand(m, n))
  Omega_vec = zeros(Int64, size(Ω,1))   # Vectorize Omega 
  for i = 1:size(Ω,1)
    Omega_vec[i] = Ω[i][1] +  size(Ω,2) * (Ω[i][2] - 1)
  end
```